mvp_onboarding.client/src/components/CustomerTable.jsx

```diff
@@ -0,0 +1,354 @@
+                name={'fitted caret up'}
+                style={{ cursor: 'pointer', opacity: sortColumn === 'Name' && sortDirection === 'asc' ? 1 : 0.5 }}
+                onClick={() => handleSort('Name', 'asc')}
+              />
+            </Table.HeaderCell>
+            <Table.HeaderCell>Address
+              &nbsp;
+              <Icon
+                name={'fitted caret down'}
+                style={{ cursor: 'pointer', opacity: sortColumn === 'Address' && sortDirection === 'desc' ? 1 : 0.5 }}
+                onClick={() => handleSort('Address', 'desc')}
+              />
+              <Icon
+                name={'fitted caret up'}
+                style={{ cursor: 'pointer', opacity: sortColumn === 'Address' && sortDirection === 'asc' ? 1 : 0.5 }}
+                onClick={() => handleSort('Address', 'asc')}
+              />
+            </Table.HeaderCell>
+            <Table.HeaderCell>Actions</Table.HeaderCell>
+            <Table.HeaderCell>Actions</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+
+        <Table.Body>
+          {customers.map((customer) => (
+            <Table.Row key={customer.id}>
+              <Table.Cell>{customer.name}</Table.Cell>
+              <Table.Cell>{customer.address}</Table.Cell>
+              <Table.Cell>
+                <Button color="yellow" onClick={() => confirmEdit(customer)}>
+                  <Icon name="edit" /> EDIT
+                </Button>
+              </Table.Cell>
+              <Table.Cell>
+                <Button color="red" onClick={() => confirmDelete(customer)}>
+                  <Icon name="trash" /> DELETE
+                </Button>
+              </Table.Cell>
+            </Table.Row>
+          ))}
+        </Table.Body>
+      </Table>
+      <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+        <Dropdown
+          compact
+          selection
+          options={pageSizeOptions}
+          value={pageSize}
+          onChange={handlePageSizeChange}
+        />
+        <Pagination
+          activePage={currentPage}
+          totalPages={Math.ceil(totalCount / pageSize)}
+          onPageChange={handlePageChange}
+          ellipsisItem={null}
+          prevItem={null}
+          nextItem={null}
+          firstItem={null}
+          lastItem={null}
+        />
+      </div>
+
+      {/* modal window for delete customer */}
+      <Modal
+        dimmer="blurring"
+        size="tiny"
+        open={deleteOpen}
+        onClose={() => setDeleteOpen(false)}
+      >
+        <Modal.Header>Delete customer</Modal.Header>
+        <Modal.Content>
+          <p>Are you sure?</p>
+        </Modal.Content>
+        <Modal.Actions>
+          <Button color="black" onClick={() => setDeleteOpen(false)}>
+            cancel
+          </Button>
+          <Button color="red" onClick={handleDelete}>
+            delete
+          </Button>
+        </Modal.Actions>
+      </Modal>
```

Comment:
This kind of setup is being used across different pages: Customer, Product, Sale, Store. Instead of creating this one by one, you can actually create a component and pass proper props to it.